include/fizzy/fizzy.h

```diff
@@ -11,8 +11,86 @@
 extern "C" {
 #endif
 
+/// The opaque data type representing a module.
+struct FizzyModule;
```

Comment:
```suggestion
typedef struct FizzyModule FizzyModule;
```

I think it will be easier to consistency use C types without `struct` prefix.